spdx-utils/build.gradle

```diff
@@ -151,12 +151,24 @@ def generateEnumClass(String taskName, String description, String jsonUrl, Strin
 def generateLicenseTextResources(String description, Map<String, Map<String, String>> ids, String resourcePath) {
```

Comment:
Just for me to double check my understanding: "generateLicenseTextResources" generates only license text file which do have an SPDX indentifier, right ?

Reply:
Correct. That includes license exceptions, though.

Reply:
Thanks!